### PR TITLE
refactored Post Entity to Content Entity

### PR DESCRIPTION
**Pull request by idchrono**
refactored Post Entity to Content Entity
Content property of Post renamed to Body
Content property of Comment renamed to Body

test data corrected for new property names
tests: passed

### CODE REVIEW

File: src/OnlineSales/Entities/Content.cs [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
+
+    public string CoverImageUrl { get; set; } = string.Empty;
+
+    public string CoverImageAlt { get; set; } = string.Empty;
+
+    [Required]
+    public string Slug { get; set; } = string.Empty;
+
+    [Required]
+    public string Template { get; set; } = string.Empty;
```
Comment (peterliapin) [inline src/OnlineSales/Entities/Content.cs:39]: Please rename Template -> Type

**Review by peterliapin — CHANGES_REQUESTED** [review]

File: src/OnlineSales/Controllers/ContentsController.cs [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
+using OnlineSales.Configuration;
+using OnlineSales.Data;
+using OnlineSales.DTOs;
+using OnlineSales.Entities;
+
+namespace OnlineSales.Controllers;
+
+[Authorize]
+[Route("api/[controller]")]
+public class ContentsController : BaseControllerWithImport<Content, ContentCreateDto, ContentUpdateDto, ContentDetailsDto, ContentImportDto>
```
[the remark this answers is not in the document]
Comment (idchrono) [reply src/OnlineSales/Controllers/ContentsController.cs]: fixed

File: src/OnlineSales/Data/PgDbContext.cs [hunk 3]
```diff
@@ -53,7 +53,7 @@ public PgDbContext(DbContextOptions<PgDbContext> options, IConfiguration configu
 
     public bool IsImportRequest { get; set; }
 
-    public virtual DbSet<Post>? Posts { get; set; }
+    public virtual DbSet<Content>? Contents { get; set; }
```
[the remark this answers is not in the document]
Comment (idchrono) [reply src/OnlineSales/Data/PgDbContext.cs]: fixed

File: src/OnlineSales/Entities/Content.cs (hunk 1 above)
Comment (idchrono) [reply src/OnlineSales/Entities/Content.cs:39]: in process

Comment (idchrono) [reply src/OnlineSales/Entities/Content.cs]: fixed

**Pull request merged**